oc-chef-pedant/spec/api/account/account_acl_spec.rb

```diff
@@ -29,7 +29,7 @@
         it "can get user acl" do
           get(request_url, platform.superuser).should look_like({
              :status => 200,
-             :body_exact => acl_body
+             :body => acl_body
```

Comment:
@stevendanna will do. The issue is that the server returns things like `[ 'pivotal', 'pivotal' ]` and cares about the order of the members of actors and groups, and the tests shouldn't care about duplicates or order. `body` lets us handle that.